src/dune_rules/coq/coq_path.mli

```diff
@@ -20,11 +20,16 @@ val path : t -> Path.t
 
 val vo : t -> Path.t list
 
+(** Unused for now *)
 val cmxs : t -> Path.t list
```

Comment:
If it is completely unused why not get rid of it?

Reply:
It could be useful in the future, for example for coqdep -modules